src/ansys/motorcad/core/geometry.py

```diff
@@ -157,6 +163,119 @@ def is_closed(self):
         else:
             return False
 
+    @property
+    def child_names(self):
+        """Property for child names list.
+
+        Returns
+        ----------
+        list of string
+            list of child region names
+        """
+        return self._child_names
+
+    def get_children(self):
```

Comment:
Worth doing the update to proper getter/setter functions? https://www.geeksforgeeks.org/getter-and-setter-in-python/